app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+
+  http_basic_authenticate_with name: "dhh", password: "secret", only: :destroy
```

Comment:
so this is kind of a quick way to do not so real authentication, with the devise gem we can streamline this functionality into a model.